extensions/ql-vscode/src/view/model-editor/ModelAlertsIndicator.tsx

```diff
@@ -24,7 +24,7 @@ export const ModelAlertsIndicator = ({
     return null;
   }
 
-  if (!evaluationRun || !modeledMethod) {
+  if (!evaluationRun?.variantAnalysis || !modeledMethod) {
```

Comment:
Why do we need to check this?

Reply:
Otherwise we might show the indicator when we don't have any alerts yet 🤔 I wanted to match what we do for the "[Evaluation run](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/src/view/model-editor/ModelEvaluation.tsx#L43-L44)" link and only show an indicator when the variant analysis has started!

Reply:
Ah okay that makes sense!